Docs/sphinx_documentation/source/BuildingAMReX.rst

```diff
@@ -28,23 +28,26 @@ list of important variables.
 
 .. table:: Important make variables
```

Comment:
follow-up PR: what's missing here and what's very important: `CC` and `CXX` variables, since environment hints are ignored by AMReX.